src/pages/TransactionDuplicate/ReviewFields.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import InteractiveStepSubHeader from '@components/InteractiveStepSubHeader';
+import SelectionList from '@components/SelectionList';
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import variables from '@styles/variables';
+import CONST from '@src/CONST';
+
+type FieldItemType = {
+    text: string;
+    value: string | boolean;
+    keyForList: string;
+};
+
+type ReviewFieldsProps = {
+    stepNames: string[];
+    label: string;
+    options: Array<{text: string; value: string | boolean}>;
+    index: number;
+    onSelectRow: (item: FieldItemType) => void;
+};
+
+function ReviewFields({stepNames, label, options, index, onSelectRow}: ReviewFieldsProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    let falsyCount = 0;
+    const filteredOptions = options.filter((name) => {
+        if (name.text !== translate('violations.none')) {
+            return true;
+        }
+        falsyCount++;
+        return falsyCount <= 1;
+    });
+    const sections = useMemo(
+        () =>
+            filteredOptions.map((option) => ({
+                text: option.text,
+                keyForList: option.text,
+                value: option.value,
+            })),
+        [filteredOptions],
+    );
+
+    return (
+        <View
+            key={index}
+            style={styles.flex1}
+        >
+            <View style={[styles.w100, styles.ph5, styles.mb5, styles.mt3, {height: CONST.BANK_ACCOUNT.STEPS_HEADER_HEIGHT}]}>
+                {stepNames.length > 1 && (
+                    <InteractiveStepSubHeader
+                        stepNames={stepNames}
+                        startStepIndex={index}
+                    />
+                )}
+            </View>
+
+            <Text
+                family="EXP_NEW_KANSAS_MEDIUM"
```

Comment:
```suggestion
                family={FontUtils.fontFamily.platform.EXP_NEW_KANSAS_MEDIUM}
```

Reply:
I think I cant use it here because for `family` prop I need to pass `FontFamilyKeys` type and this what you provided its just a string with font family name which in this case is `ExpensifyNewKansas-Medium, Segoe UI Emoji, Noto Color Emoji`

Reply:
I got you 🙂